app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find(params[:user_id])
+    post = Post.find(params[:post_id])
+    text = params[:text]
+
+    new_comment = Comment.new(text:, post:, author: user)
+
+    if new_comment.save
+      redirect_to user_post_url(user, post)
+    else
+      render :new
+    end
+  end
```

Comment:
- [OPTIONAL] It will be nice to add strong parameters to your controllers as this will make your app more secure. Something like so;
```
    private 
    def comments_params
      params.require(:comment).permit(:text)
    end
```

Reply:
@t-yanick 

Can you provide some resources on how to use strong parameters